CHANGELOG.md

```diff
@@ -4,6 +4,23 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 8.3.1
+
+### Important Changes
+
+- **feat(nextjs): Trace pageloads in App Router (#12157)**
+
+If you are using Next.js version `14.3.0-canary.64` or above, the Sentry Next.js SDK will now trace clientside pageloads
+with React Server Components. This means, that client-side errors like
+`Error: An error occurred in the Server Components render.`, which previously didn't give you much information on how
+that error was caused, can now be traced back to a specific error in a server component.
+
+### Other Changes
+
+- feat(angular): Add Support for Angular 18 (#12183)
```

Comment:
this means it should be a minor `8.4.0`. We should also mark angular support as an important change.

Reply:
> this means it should be a minor

Why is that? We didn't add any API in this release.

Reply:
Ah, I guess since it is not safe to revert